scss/components/_dropdowns.scss

```diff
@@ -0,0 +1,118 @@
+// Dropdowns
+//
+// Default styling is for the filter panels
+//
+// Markup:
+// <fieldset class="js-checkbox-filters">
+//   <legend class="label">Election Years</legend>
+//   <ul class="dropdown__selected"></ul>
+//   <div class="dropdown">
+//     <button class="js-toggle js-dropdown button--dropdown button--neutral" data-toggles="cycle-dropdown">More</button>
+//     <div id="cycle-dropdown" class="dropdown__panel" aria-hidden="true">
+//       <ul class="dropdown__list">
+//         <li>
+//           <input id="cycle-checkbox-year" name="cycle" type="checkbox" value="2011" />
+//           <label class="dropdown__value" for="cycle-checkbox-year">2011 - 2012</label>
+//         </li>
+//       </ul>
+//     </div>
+//   </div>
+// </fieldset>
+// 
+// Styleguide components.dropdowns
+
+.button--dropdown {
```

Comment:
I go back on forth with this but I feel like if this is in the dropdown file, it should be `dropdown--button`, `dropdown__button`.